packages/typespec-ts/package.json

```diff
@@ -65,7 +65,7 @@
     "@azure-tools/cadl-ranch-expect": "^0.13.2",
     "@azure-tools/cadl-ranch": "^0.12.4",
     "chalk": "^4.0.0",
-    "@azure-rest/core-client": "^1.3.0",
+    "@azure-rest/core-client": "^1.4.0",
```

Comment:
do we need to bump client version in generated code?

Reply:
@v-jiaodi can you help send a fix to update that version ?